web/src/routes/EventsPage.tsx

```diff
@@ -112,7 +123,9 @@ const EventsPage = ({ cityModel, languages, pathname, languageCode, cityCode }:
           content={content}
           title={title}
           formatter={formatter}
-          onInternalLinkClick={navigate}>
+          onInternalLinkClick={navigate}
+          buttonText={t('exportAsICal')}
```

Comment:
i'm not super happy with this. Maybe we could just pass a `pageFooter` prop (ReactNode) and pass the
`<TextButton/>` directly.  Because it makes the page less generic and maybe we want to display more information in the footer later on

Reply:
What do you mean by this? Perhaps something like described in this article? https://medium.com/trabe/passing-elements-as-props-in-react-54f9841ec0c9 
`const Confirm = ({
  children,
  onAccept,
  onReject,
  renderAcceptButton = onAccept => (
    <Button className="accept-btn" onClick={onAccept}>
      OK
    </Button>
  ),
  renderRejectButton = onReject => (
    <Button className="reject-btn" onClick={onReject}>
      Cancel
    </Button>
  ),
}) => (
  <div className="confirm">
    <div className="confirm-header">
      <h1>Confirm</h1>
    </div>
    <div className="confirm-content">{children}</div>
    <div className="confirm-footer">
      {renderAcceptButton(onAccept)}
      {renderRejectButton(onReject)}
    </div>
  </div>
);


<Confirm
  renderAcceptButton={onAccept => (
    <Button className="accept-btn" onClick={onAccept}>
      Yep
    </Button>
  )}
  renderRejectButton={onReject => (
    <Button className="cancel-btn" onClick={onReject}>
      Nope
    </Button>
  )}
>
  You sure?
</Confirm>`

Reply:
Exactly, pass `<TextButton text=... />` instead of the props of text button. That way, the Page component stays reusable and doesn't need to know implementation details.

Reply:
I tried to do that but I got an error telling me that I'm passing a JsxElement but not a TextButton...imo it's the best if someone just finishs the Issue.  This possibly takes as long as helping me undestanding that error